### ISSUE TITLE
How to restore from lastly saved checkpoint?

### ISSUE THREAD

**Issue by abeyang00** [wizyoung/YOLOv3_TensorFlow#6]
Whenever i try to restore from lastly saved checkpoint, code seems to train from scratch again.

I don't restore from yolo.ckpt (from COCO dataset) after training.

**Comment by wizyoung** [comment]
just change the argument `--restore_part` to `[None]`.

**Comment by abeyang00** [comment]
Thank you. Also after 20~30 Epochs my loss becomes NaN.

Does that mean training is done or my detector is stuck in local minima? 
should i decrease learning rate?

**Comment by abeyang00** [comment]
Thank you!

**Comment by abeyang00** [comment]
@wizyoung 

Actually, when i changed the argument --restore_part to [None].
I get an error like this 

<img width="933" alt="screen shot 2019-01-27 at 11 30 33 pm" src="https://user-images.githubusercontent.com/36392866/51802267-affe7c00-228b-11e9-8630-f164eb6a7ce0.png">

Am i supposed to also change checkpoint_dir or restore_path?

**Comment by wizyoung** [comment]
yes, change the restore_path
